vm/systemSmartContracts/staking.go

```diff
@@ -753,7 +753,8 @@ func (s *stakingSC) unBond(args *vmcommon.ContractCallInput) vmcommon.ReturnCode
 	}
 
 	currentNonce := s.eei.BlockChainHook().CurrentNonce()
-	if registrationData.UnStakedNonce > 0 && currentNonce-registrationData.UnStakedNonce < s.unBondPeriod {
+	isUnstakedWaiting := registrationData.UnStakedNonce == core.DefaultUnstakedEpoch
```

Comment:
nonce is compared with epoch: registrationData.UnStakedNonce == core.DefaultUnstakedEpoch ?